go/examples/vaccine_test.go

```diff
@@ -0,0 +1,142 @@
+package examples
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	wallet "github.com/trinsic-id/sdk/go/proto/universalwallet/v1"
+	template "github.com/trinsic-id/sdk/go/proto/verifiablecredentials/templates/v1"
+	credential "github.com/trinsic-id/sdk/go/proto/verifiablecredentials/v1"
+	"github.com/trinsic-id/sdk/go/services"
+	"testing"
+)
+
+func createTemplate(templateService services.CredentialTemplateService, t *testing.T) *template.TemplateData {
+	// createTemplate() {
+	templateRequest := &template.CreateCredentialTemplateRequest{Name: "VaccinationCertificate", AllowAdditionalFields: false, Fields: make(map[string]*template.TemplateField)}
+	templateRequest.Fields["firstName"] = &template.TemplateField{Description: "Given name"}
+	templateRequest.Fields["lastName"] = &template.TemplateField{}
+	templateRequest.Fields["batchNumber"] = &template.TemplateField{Type: template.FieldType_NUMBER}
```

Comment:
```suggestion
	templateRequest.Fields["batchNumber"] = &template.TemplateField{Description: "Batch number of vaccine"}
```

Batch number is just a string for right now (I figured batch numbers wouldn't be solely decimal), so for template parity, this should be a string.

However, this makes me think that we should change the type to a number in all languages to demonstrate the capability.

Reply:
I vote change to number, that was the original intent anyway.

Reply:
> I vote change to number, that was the original intent anyway.

Roger that -- we'll leave it as a number here, and I'll track the changes to the other langs.

Reply:
Updated the suggestion